<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuySellCCWidget</class>
 <widget class="QWidget" name="BuySellCCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="buySellCcFormLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="quantityLabel">
       <property name="text">
        <string>&amp;Quantity</string>
       </property>
       <property name="buddy">
        <cstring>lineEditQuantity</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="quantityWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEditQuantity"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMax">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEstimated">
       <property name="text">
        <string>Estimated Value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lineEditEstimated">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>yyy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="receivingWalletLabel">
       <property name="text">
        <string>&amp;Wallet</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxWallets</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="receivingWalletWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBoxWallets">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAdvanced">
          <property name="text">
           <string>Advanced</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="cashBalanceLabel">
       <property name="text">
        <string>Balance</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelBallanceValue">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>xxx</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="receivingAddressLabel">
       <property name="text">
        <string>&amp;Receiving Address</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxReceivingAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxReceivingAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
